content/en/blog/2019/istioctl-describe/index.md

```diff
@@ -0,0 +1,244 @@
+---
+title: "'istioctl describe' in Istio 1.3"
+description: Introducing a command-line utility to find configuration affecting pod traffic.
+publishdate: 2019-09-20
+attribution: Ed Snible (IBM)
+keywords: [traffic-management, istioctl, debugging]
+---
+
+Istio 1.3 includes an experimental command-line tool designed to help find and
+understand configuration that affects traffic to a pod.
```

Comment:
There is nothing wrong with your use but `affect` and `effect` can be very confusing for non-English native readers. Consider:
```suggestion
understand the configuration that impacts the traffic to a pod.
```